src/rust/zh.manhuagui/src/lib.rs

```diff
@@ -12,6 +12,12 @@ use aidoku::{
 
 const BASE_URL: &str = "https://www.manhuagui.com";
 
+#[initialize]
+fn initialize() {
+	aidoku::std::net::set_rate_limit(5);
+	aidoku::std::net::set_rate_limit_period(10);
```

Comment:
works fine!

Reply:
aidoku's rate limit api is not the greatest

Reply:
i thought the remained requests would start after `set_rate_limit_period()`, but they seemed to be dropped instead?
i should keep this pr only for nsfw change and sorting methods

Reply:
it just prevents the request from going through to the site and returns an error to prevent being rate limited / banned by the site. it's meant for hard cut offs where users will be banned if they exceed a certain amount of requests.